src/typings/charts/line-chart.d.ts

```diff
@@ -1,109 +1,113 @@
-import { ChartBaseAPI, InteractiveChartAPI, ExportableChartAPI } from '../common/base';
+import {
+    ChartBaseAPI,
+    InteractiveChartAPI,
+    ExportableChartAPI,
+    ThemableChartAPI,
+} from '../common/base';
 import { GridTypes } from '../common/grid';
 import { ChartModuleSelection } from '../common/selection';
 import { AxisTimeCombination } from '../helpers/constants';
 
-
 export enum LineChartKeys {
-  Date = 'date',
-  Name = 'name',
-  TopicName = 'topicName',
-  Value = 'value',
+    Date = 'date',
+    Name = 'name',
+    TopicName = 'topicName',
+    Value = 'value',
 }
 
 export type LineChartDataShape = {
-  [LineChartKeys.Date]: string;
-  [LineChartKeys.Name]: string;
-  [LineChartKeys.TopicName]: string;
-  [LineChartKeys.Value]: number;
+    [LineChartKeys.Date]: string;
```

Comment:
is the formatting still preserved? I assume it follows the eslint rules or Prettier configs we set

Reply:
Prettier, yes.
We only lose the `.append` thing, but that gave more work than what was worth it!